engine/apps/api/views/alert_receive_channel.py

```diff
@@ -29,26 +28,11 @@
     TeamFilteringMixin,
     UpdateSerializerMixin,
 )
+from common.api_helpers.paginators import FifteenPageSizePaginator
 from common.exceptions import MaintenanceCouldNotBeStartedError, TeamCanNotBeChangedError, UnableToSendDemoAlert
 from common.insight_log import EntityEvent, write_resource_insight_log
 
 
-class AlertReceiveChannelPagination(PageNumberPagination):
-    page_size = 15
-    page_query_param = "page"
-    page_size_query_param = "perpage"
-    max_page_size = 50
-
-    def paginate_queryset(self, queryset, request, view=None):
-        """Override to apply pagination only if ?page= is present in query params
-        Required for backwards compatibility with older versions
-        """
-        page_number = request.query_params.get(self.page_query_param, None)
-        if not page_number:
-            return None
-        return super().paginate_queryset(queryset, request, view)
```

Comment:
I think this can be removed. @iskhakov wdyt?

This was _likely_ here to handle the transition when the paginated integration's page was release, and to handle the case in Cloud where UI clients running the previous version would've received a different API response.

Reply:
I agree